src/acl/group.test.ts

```diff
@@ -801,8 +807,8 @@ describe("getGroupResourceAccessAll", () => {
     resourceAcl.add(
       DataFactory.quad(
         getLocalNode(agentClassRuleSubjectIri),
-        DataFactory.namedNode("http://www.w3.org/ns/auth/acl#agent"),
-        DataFactory.namedNode("https://some.pod/agent#webId")
+        DataFactory.namedNode("http://www.w3.org/ns/auth/acl#agentGroup"),
+        DataFactory.namedNode("https://some.pod/groups#group")
```

Comment:
Are you sure this change is relevant? Reading the test description the idea is that the Agent-specific Quads do not affect the return value, but if there are none, then of course they're not going to affect the return value anyway :)

Reply:
You are absolutely right ^^
Fixed in c6218eef22bc3c5264aca8cf6fa2347dead169b4